packages/components/package.json

```diff
@@ -62,6 +62,8 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "build": "rollup -c --bundleConfigAsCjs",
+    "preversion": "#yarn test",
```

Comment:
What is this doing?

Reply:
That is a future placeholder for running tests once we have tests written. All I would need to do is uncomment the script and it would run tests before publishing. If the tests fail they would fail the PR checks